spec/spec_helper.rb

```diff
@@ -57,7 +61,11 @@ class Application < Rails::Application
   # drop the database after the suite.
   config.after(:suite) do
     if ENV["CI"]
-      Mongoid::Threaded.sessions[:default].drop
+      if Mongoid.respond_to?(:default_client)
```

Comment:
Instead of this, might want consider using [mongoid-compatibility](https://github.com/dblock/mongoid-compatibility) and one Gemfile with difference ENV for Travis to test against multiple mongoid versions (that gem is a good demo of what that looks like).